src/base/utils/misc.cpp

```diff
@@ -625,8 +626,15 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath)
     if ((output == "dolphin.desktop") || (output == "org.kde.dolphin.desktop"))
         proc.startDetached("dolphin", QStringList() << "--select" << Utils::Fs::toNativePath(path));
     else if ((output == "nautilus.desktop") || (output == "org.gnome.Nautilus.desktop")
-             || (output == "nautilus-folder-handler.desktop"))
-        proc.startDetached("nautilus", QStringList() << "--no-desktop" << Utils::Fs::toNativePath(path));
+             || (output == "nautilus-folder-handler.desktop")) {
+        proc.start("nautilus", QStringList() << "--version"); //fixes #8923
+        proc.waitForFinished();
+        QString nautilusV = proc.readLine().simplified();
+        if (nautilusV.remove(QRegExp("[^0-9]")) >= QString("3280"))
```

Comment:
`QLatin1String("3280")`

Reply:
that about 
``
if (nautilusVer.remove(QRegExp("[^0-9]")).toInt(&ok, 10) >= 3280)
``

Reply:
and compare it:
`nautilusVer < Utils::Version<int, 3>("3.28.0")`